.circleci/config.yml

```diff
@@ -157,6 +157,84 @@ jobs:
       - store_test_results:
           path: make-recall-decision-e2e-tests/e2e_tests/junit
 
+  integration_tests:
+    machine:
+      image: ubuntu-2204:current
+    parallelism: << pipeline.parameters.e2e-parallelism >>
+    steps:
+      - run:
+          name: Clone and checkout make-recall-decision-e2e-tests
+          command: |
+            git clone https://github.com/ministryofjustice/make-recall-decision-e2e-tests.git
+            cd make-recall-decision-e2e-tests 
+            git checkout $CIRCLE_BRANCH
+      - run:
+          name: Clone and checkout make-recall-decision-api
+          command: |
+            git clone https://github.com/ministryofjustice/make-recall-decision-api.git
+            cd make-recall-decision-api
+            git checkout main
+      - run:
+          name: Clone and checkout make-recall-decision-ui
+          command: |
+            git clone https://github.com/ministryofjustice/make-recall-decision-ui.git
+            cd make-recall-decision-ui
+            git checkout main
+      - run:
+          name: Start components for integration tests
+          working_directory: make-recall-decision-e2e-tests
+          command: ./scripts/start-services-integration.sh
+      - run:
+          name: Run Integration Tests
+          working_directory: make-recall-decision-e2e-tests
+          command: |
+            npm ci --no-audit
+            echo PWD:: $PWD
+            shopt -s globstar
+
+            echo Generating specs
+            SPECS=$(circleci tests glob "e2e_tests/features/**/*.feature" | circleci tests split --split-by=timings --show-counts)
+            echo "Running feature file(s): $SPECS"
+
+            set +e
+
+            npx cypress run \
+              --env=TAGS='not @E2E and not @smoke',USERNAME_PO=${int_USERNAME_PO},PASSWORD_PO=${int_PASSWORD_PO},USERNAME_SPO=${int_USERNAME_SPO},PASSWORD_SPO=${int_PASSWORD_SPO},USERNAME_ACO=${int_USERNAME_ACO},PASSWORD_ACO=${int_PASSWORD_ACO},USERNAME_PPCS=${int_USERNAME_PPCS},PASSWORD_PPCS=${int_PASSWORD_PPCS},API_CLIENT_ID=${int_API_CLIENT_ID},API_CLIENT_SECRET=${int_API_CLIENT_SECRET},MAKE_RECALL_DECISION_API_URL=${int_MAKE_RECALL_DECISION_API_URL},HMPPS_AUTH_EXTERNAL_URL=${int_HMPPS_AUTH_EXTERNAL_URL} \
```

Comment:
A note that, since I was founding a new script so we can control this pipeline from it's own repo, I have moved to using the test scope (in this case `int_` for integration) as a prefix. This doesn't matter here but has proven helpful on CircleCI where [environmental variables are sorted alphabetically](https://app.circleci.com/settings/project/github/ministryofjustice/make-recall-decision-e2e-tests/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fministryofjustice%2Fmake-recall-decision-e2e-tests%2F167%2Fworkflows%2Fdff4668b-89b7-4571-9d42-56ba9fc6dc50%2Fjobs%2F195) so has the effect of collecting the variables for this scope together.
No other variables yet been removed and will not until we know this is working in parallel over on the ui pipelines.